restapi/src/main/java/io/stargate/web/resources/v2/RowsResource.java

```diff
@@ -693,18 +692,17 @@ private Object getRows(
     return raw ? rows : new GetResponseWrapper(rows.size(), newPagingState, rows);
   }
 
-  private List<ColumnOrder> buildSortOrder(String sort)
-      throws com.fasterxml.jackson.core.JsonProcessingException {
+  private List<ColumnOrder> buildSortOrder(String sort) {
     if (Strings.isNullOrEmpty(sort)) {
       return new ArrayList<>();
     }
 
     List<ColumnOrder> order = new ArrayList<>();
-    Map<String, String> sortOrder = mapper.readValue(sort, Map.class);
+    Map<String, Object> sortOrder = ResourceUtils.readJson(sort, mapper);
```

Comment:
where the `IllegalArgumentException` thrown from this method is handled?

Reply:
Here
https://github.com/stargate/stargate/blob/master/restapi/src/main/java/io/stargate/web/resources/RequestHandler.java#L48